### PR TITLE
Swap default image to Minimal 

### LINKED ISSUE #146: Swap default image to Minimal
Currently the default image is the normal tag, moving forward it would be more convenient to have minimal as the default and normal as the option.

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Fixes #146

### CODE REVIEW

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ metadata:
           },
           "spec": {
             "operand": {
-              "image": "gcr.io/k8s-staging-nfd/node-feature-discovery:master",
+              "image": "k8s.gcr.io/nfd/node-feature-discovery:v0.11.0-minimal",
```
Comment (marquiz) [inline config/manifests/bases/nfd-operator.clusterserviceversion.yaml:16]: Think we should use master-minimal here.

**Review by marquiz — COMMENTED** [review]
Thanks @ArangoGutierrez. Does documentation need any updating?

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (ArangoGutierrez) [reply config/manifests/bases/nfd-operator.clusterserviceversion.yaml:16]: ok

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -463,8 +463,8 @@ spec:
                 - name: OPERATOR_NAME
                   value: cluster-nfd-operator
                 - name: NODE_FEATURE_DISCOVERY_IMAGE
-                  value: k8s.gcr.io/nfd/node-feature-discovery:master
-                image: k8s.gcr.io/nfd/node-feature-discovery-operator:master
+                  value: k8s.gcr.io/nfd/node-feature-discovery:v0.11.0-minimal
```
Comment (marquiz) [inline config/manifests/bases/nfd-operator.clusterserviceversion.yaml:466]: Needs to be master, too
```suggestion
                  value: k8s.gcr.io/nfd/node-feature-discovery:master-minimal
```

Comment (ArangoGutierrez) [reply config/manifests/bases/nfd-operator.clusterserviceversion.yaml:466]: This registry doesn't hold a master tag. Only staging has a master tag.
Will make the change and push

**Pull request merged**